modules/bringauto_repository/GitLFSRepository.go

```
@@ -43,40 +50,155 @@
+	return nil
+}
+
+func printErrors(errorPackPaths []string, expectedPackForImagePaths []string, expectedPackNotForImagePaths []string) error {
+	logger := bringauto_log.GetLogger()
+	if len(errorPackPaths) > 0 {
+		logger.Error("%d packages are not in Json definitions but are in Git Lfs (listing first %d):", len(errorPackPaths), listFileCount)
+		for i, errorPackPath := range errorPackPaths {
+			if i > listFileCount - 1 {
+				break
+			}
+			logger.ErrorIndent(errorPackPath)
```

golangci-lint (1.62.2)

printf: non-constant format string in call to (*bringauto/modules/bringauto_log.Logger).ErrorIndent

(govet)